app/assets/stylesheets/partners.scss

```diff
@@ -4,3 +4,16 @@
   margin: 2em;
 }
 
+ul.donors {
+  text-align: center;
+  margin: 3em 0 6em 0;
+
+  li {
+    display: inline;
+    margin: 0 10em 0 0;
+
+    &:last-child {
+      margin: 0;
+    }
+  }
+}
```

Comment:
@jasonoverby Just need to wrap in a page-specific div like this, so that styles applied to `ul` elements don't trump the ones used elsewhere (like in the nav). Also: SCSS lets you nest, which I find nicer to read, though that's a matter of taste.